aiowebostv/webos_client.py

```diff
@@ -96,28 +98,37 @@ def registration_msg(self):
         handshake["payload"]["client-key"] = self.client_key
         return handshake
 
+    async def _ws_connect(self, uri, ssl_context):
+        """Create websocket connection."""
+        _LOGGER.debug("connect(%s): uri: %s", self.host, uri)
+        return await ws_connect(
+            uri,
+            ping_interval=self.ping_interval,
+            ping_timeout=self.ping_timeout,
+            open_timeout=self.timeout_connect,
+            close_timeout=self.timeout_connect,
+            max_size=None,
+            ssl=ssl_context,
+        )
+
     async def connect_handler(self, res):
         """Handle connection for webOS TV."""
         # pylint: disable=too-many-locals,too-many-statements
         handler_tasks = set()
         main_ws = None
         input_ws = None
+        ssl_context = None
         try:
-            # webOS uses self-signed certificates, thus we need to use an empty
-            # SSLContext to bypass validation errors.
-            ssl_context = ssl.SSLContext()
-
-            main_ws = await asyncio.wait_for(
```

Comment:
`asyncio.wait_for` is implemented by setting `open_timeout`:
https://github.com/aaugustin/websockets/blob/75bb1cb07a476e899b689c5f50872f90f98a38e5/src/websockets/legacy/client.py#L659